styles/main.css

```diff
@@ -59,37 +59,42 @@ main #useful-links a{
 footer {
     text-align: center;
     font-size: 12px;
-    margin: 0 0 0 3em;
+    margin: 0 0 3em 0;
 }
 
 .width-wrap {
     background-color: #7EAEE4;
-    border: 1px groove rgb(0,0,0,0.3);
-    width:1024px;

```

Comment:
Use `max-width` instead to achieve the no-scrollbars look for windows less than 1024px wide